### PR TITLE
fix: ui bugs in apps

### LINKED ISSUE #5461: [PRO-26] event-type-settings nav UI issues
The icons are misaligned in the sidebar nav

> From [SyncLinear.com](https://synclinear.com) | [PRO-26](https://linear.app/calcom/issue/PRO-26/event-type-settings-nav-ui-issues)

### PR DESCRIPTION
**Pull request by Udit-takkar**
Fixes: #5461 , #5460 

https://user-images.githubusercontent.com/53316345/201128487-3c6ee461-8884-427e-af62-a8a947e5e99a.mov

### CODE REVIEW

**Comment by Udit-takkar** [comment]
Screenshot 2022-11-10 at 9 10 22 PM

File: packages/app-store/_components/AppCard.tsx [hunk 1]
```diff
@@ -26,35 +26,40 @@ export default function AppCard({
...
+      <div className="p-4 text-sm sm:p-8">
+        <div className="flex w-full flex-col gap-2 sm:flex-row sm:gap-0">
+          {/* Don't know why but w-[42px] isn't working, started happening when I started using next/dynamic */}
+          <Link href={"/apps/" + app.slug}>
+            <a className="mr-3 h-auto w-10 rounded-sm">
+              <img src={app?.logo} alt={app?.name} />
+            </a>
+          </Link>
+          <div className="flex flex-col">
+            <span className="font-semibold leading-none text-black">{app?.name}</span>
```
Comment (CarinaWolli) [inline packages/app-store/_components/AppCard.tsx:39]:
```suggestion
            <span className="text-base leading-4 font-semibold leading-none text-black">{app?.name}</span>
```

File: packages/app-store/_components/AppCard.tsx [hunk 2]
```diff
@@ -26,35 +26,40 @@ export default function AppCard({
...
+            <p className="pt-2 text-sm font-normal text-gray-600">{description || app?.description}</p>
```
Comment (CarinaWolli) [inline packages/app-store/_components/AppCard.tsx:40]:
```suggestion
            <p className=" leading-4 pt-2 text-sm font-normal text-gray-600">{description || app?.description}</p>
```

**Review by CarinaWolli — CHANGES_REQUESTED** [review]
The image size is still not 40px. Width and height of 28px is coming from the SVG file 
Screenshot 2022-11-11 at 09 52 32

File: packages/app-store/_components/AppCard.tsx (hunk 1 above)
Comment (Udit-takkar) [reply packages/app-store/_components/AppCard.tsx]: done

File: packages/app-store/_components/AppCard.tsx (hunk 2 above)
Comment (Udit-takkar) [reply packages/app-store/_components/AppCard.tsx]: done

**Comment by Udit-takkar** [comment]
Screenshot 2022-11-11 at 4 48 24 PM

@CarinaWolli  stripe one is fixed but there are still few of the logos(fathom and google analytics) which are not 40px . don't really know how to fix this. tried setting both height and width to 40px but it still resize.

**Review by CarinaWolli — APPROVED** [review]
Looks good to me now! The logo size seems to be fixed too 🙏

**Review by PeerRich — APPROVED** [review]

**Pull request merged**